src/core/test_helpers/kbn_server.ts

```diff
@@ -9,12 +9,19 @@
 import { Client } from 'elasticsearch';
 import { ToolingLog, REPO_ROOT } from '@kbn/dev-utils';
 import {
+  // @ts-expect-error
   createLegacyEsTestCluster,
+  // @ts-expect-error
   DEFAULT_SUPERUSER_PASS,
+  // @ts-expect-error
   esTestConfig,
+  // @ts-expect-error
   kbnTestConfig,
+  // @ts-expect-error
   kibanaServerTestUser,
+  // @ts-expect-error
   kibanaTestUser,
+  // @ts-expect-error
```

Comment:
Why was this needed?

Reply:
Previously we were importing the types from the source and we had those types available. Right now we are importing from the build assets and we don't have those types anymore as we were able to generate them to avoid the type clashes between jest and mocha. Spencer opened an issue with a way we define to overcome this problem (https://github.com/elastic/kibana/issues/95679) but as it requires some development time we won't use it as a solution part of this PR.